test/end-to-end/pages/CreateCase.js

```diff
@@ -119,6 +119,11 @@ module.exports = {
     await I.retry(retryCount).click('#caseTypeOfApplication-FL401');
     await I.retry(retryCount).click('Continue');
   },
+  async selectTypeOfApplicationFL401_TS() {
```

Comment:
@gokul-sol , is this test case required . please check and modify it accordingly.

Reply:
@yogendraU this is a helper test step we might use to create test cases. Yes, it is needed.